javascript/extractor/src/com/semmle/js/extractor/AutoBuild.java

```diff
@@ -685,7 +687,8 @@ public int compare(Path f1, Path f2) {
    * Like {@link #PATH_ORDERING} but for {@link File} objects.
    */
   public static final Comparator<File> FILE_ORDERING = new Comparator<File>() {
-    public int compare(File f1, File f2) {
+    @Override
+	public int compare(File f1, File f2) {
```

Comment:
This indentation looks weird?

Reply:
Thanks, fixed.  Out of interest, how do you tend to edit these files?  There doesn't seem to be a formatter configured in Eclipse.